compiler/rustc_lint/src/non_fmt_panic.rs

```diff
@@ -67,7 +79,11 @@ impl<'tcx> LateLintPass<'tcx> for NonPanicFmt {
     }
 }
 
-fn check_panic<'tcx>(cx: &LateContext<'tcx>, f: &'tcx hir::Expr<'tcx>, arg: &'tcx hir::Expr<'tcx>) {
+fn check_panic_or_unreachable<'tcx>(
```

Comment:
This function also covers `debug_assert` and `assert`, but since `check_panic_or_assert_or_debug_assert_or_unreachable` gets a bit long, I think just `check_panic` suffices, as they're all forms of panic.